### ISSUE TITLE
[FEATURE REQUEST] Alternative layout option for Orbit clock widget

### ISSUE THREAD

**Issue by straycosmicllama** [MM2-0/Kvaesitso#747]
Current orbit clock widget's layout is [seconds(small), minutes(medium), hours(big)] (inner to outer).
Is it possible to add an option for alternative layout like [hours(big), minutes(medium), seconds(small)] (inner to outer)?

**Comment by Fred6388** [comment]
I completely agree, the object with the greatest mass will always be the center of gravity. lack some basic physics here.

**Comment by strongville** [comment]
IMHO the intention for the Orbit clock style is that would be as readable as a hands clock as possible, inverting the position (more noticeable with the small style variant) would make it unreadable if you don't have visible the numbers to back it up.

**Comment by Sir-Photch** [comment]
"Basic physics" dictate that objects closer to the center of mass move faster.

**Comment by derei** [comment]
> I completely agree, the object with the greatest mass will always be the center of gravity. lack some basic physics here.

Tell that to Jupiter, Saturn, Uranus... they didn't get the memo when Solar System was formed and they are wandering quite far... 😄

**Comment by exstntlmsnthrp** [comment]
Ummm, I'm with @Sir-Photch on this. Mars definitely revolves around the sun much faster than Jupiter, Uranus, or anything else further out there.

**Comment by freijon** [comment]
I was about to create a new feature request for this and found this issue. I agree with the requester. I don't really know why, but it somehow feels weird to me. I LOVE the idea in principal but not the order of the "planets" :D

I think it's because on the seconds I'd like to see the position (angle) most accurately (put it on the most outward ring) and the hours turn so slow it doesn't matter when you don't see the angle very good. Or maybe its because on analog watches the hour hand is the shortest, followed by minutes and then seconds. But I guess it's just my subjective readability preference and it doesn't consider at all how planetary bodies move in reality :D

Also, I'd love to make the entire widget bigger.
